app/.eslintrc.js

```diff
@@ -52,8 +53,19 @@ module.exports = {
         }
       }
     ],
+    'jsx-quotes': ['error', 'prefer-double'],
```

Comment:
@JannikWibker should we really define both types of quotes (take a look at line #18)?

Reply:
The one above is for JS/TS quotes, the one below for JSX. As JSX mirrors HTML in syntax the standard is to use double quotes. Thus I'd say defining both is a good thing.

This is also the default in many other projects, even when it is not necessarily enforce by linting rules. As a side note the default in prettier is also to use double quotes in JSX.

Reply:
Makes sense, lets stick with the standards 👍🏻